lib/modules/platform/github/index.ts

```diff
@@ -1318,15 +1324,29 @@ export async function ensureCommentRemoval(
 
 // Pull Request
 
-async function tryPrAutomerge(
+export async function tryPrAutomerge(
```

Comment:
i don't like to see this exported. Can we instead test the caller like we to with toher tests?